java/src/lib.rs

```diff
@@ -15,8 +16,93 @@ mod ffi_test;
 #[cfg(ffi_test)]
 pub use ffi_test::*;
 
+#[allow(dead_code)]
+/// Create a `JObject` from Rust's `String`. In case of an error
+/// this method throws Java exception and returns `JObject::null`
+fn create_glide_string_from_string<'local>(
```

Comment:
You don't need to call `GlideString of(String)` from rust, because `GlideString of(byte[])` only used there.

Reply:
I am going to keep it here (removed if not used). I am still not sure if we want to change Simple & Verbatim string. Its because how the Java code is written: we have `handleStringResponse()`, but this method does not differentiate between the type of the string. I will do code cleanup once we will have all commands covered